README.md

```diff
@@ -155,6 +155,35 @@ Which returns:
 <span title="my title">Hello, World!</span>
 ```
 
+##### Validation
+
+Components inherited from `ActionView::Component::Base` extends with `ActionModel::Validations` by default. You can skip this behaviour by config:
+
+`config/application.rb`:
+```ruby
+config.action_view_component.validation_module = 'MyValidation'
+```
+
+You can define your own validation mechanism. `validation_module` should be set to the string that represents module name. Validation module has to provide `validate!` method:
+
+`lib/my_validation.rb`
+```ruby
+module MyValidation do
+  def validate!
+    # some validation checks here
+  end
+end
+```
+
+You can also skip validation in some components with `skip_validation`:
+
+`app/components/validation_disabled.rb`:
+```ruby
+class ValidationDisabled < ActionView::Component::Base
+  self.skip_validation = true
+end
+```
```

Comment:
I don't see the need for both mechanisms and its confusing to provide the choice. Most folks will stick wit h the built in ActiveModel Validations. If you want finer control you supply your own module. Users are free to build features like `skip_validation` into their own validation module but I don't think we need it out of the box.